routers/web/admin/emails.go

```diff
@@ -109,6 +116,13 @@ func isKeywordValid(keyword string) bool {
 
 // ActivateEmail serves a POST request for activating/deactivating a user's email
 func ActivateEmail(ctx *context.Context) {
+
```

Comment:
```suggestion
```

Reply:
Done in 25673a937efd78833756f298bfd344b214158151.